static/scripts/content/config.js

```diff
@@ -0,0 +1,21 @@
+// mymodule.js
+module.exports = {
+  configs: {
+    API: {
+      baseUrl: 'http://localhost', //'https://schul-cloud.org',
```

Comment:
📛

Reply:
please use ENV variables with defaults

Reply:
Yes of course I will do so. I justed marked this here so we don't forget it. By the way: Do you know whats the name of the env variable that could be used here? it isn't `BACKEND_URL`. And do we have an overview over all ENV-Variables anywhere?

Reply:
@janrenz @adrianjost You're right, atm we don't have an overview for all envs. For the Content-Services we're proxy-ing all traffic through the server, as can be viewed in the current implementation. BUT for N21 backendURL != cotentBackendURL. I'ld sugget using `CONTENT_URI`, see https://github.com/schul-cloud/schulcloud-server/blob/master/config/production.json.